cli/network.go

```diff
@@ -234,3 +236,71 @@ EnableIPV6:   false
 ID:           c33c2646dc8ce9162faa65d17e80582475bbe53dc70ba0dc4def4b71e44551d6
 Internal:     false`
 }
+
+// networkListDescription is used to describe network list command in detail and auto generate command doc.
+var networkListDescription = "List networks in pouchd. " +
+	"It lists the network's Id, name, driver and scope."
+
+// NetworkListCommand is used to implement 'network list' command.
+type NetworkListCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkListCommand command.
+func (n *NetworkListCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List pouch networks",
+		Long:    networkListDescription,
+		Args:    cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return n.runNetworkList(args)
+		},
+		Example: networkListExample(),
+	}
+
+	n.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (n *NetworkListCommand) addFlags() {
+	//TODO add flags
+}
+
+// runNetworkList is the entry of NetworkListCommand command.
+func (n *NetworkListCommand) runNetworkList(args []string) error {
+	logrus.Debugf("list the networks")
+
+	apiClient := n.cli.Client()
+	resp, err := apiClient.NetworkList()
+	if err != nil {
+		return err
+	}
+
+	display := n.cli.NewTableDisplay()
+	display.AddRow([]string{"NETWORK ID", "NAME", "DRIVER", "SCOPE"})
+	for _, network := range resp.Networks {
+		display.AddRow([]string{
+			network.ID,
```

Comment:
change the id to short id is better？

Reply:
I think short ID is better, since the original ones is too long. And it may make output in cli not so readable.

Reply:
I agree with you two.